app/docker/views/containers/create/createContainerController.js

```diff
@@ -565,6 +566,21 @@ function ($q, $scope, $state, $timeout, $transition$, $filter, Container, Contai
       Notifications.error('Failure', err, 'Unable to retrieve networks');
     });
 
+    SystemService.info()
```

Comment:
There is already a `SystemService.info()` call right below the `Container.query` call, you can process the runtime information in the success function there instead of making another call to `SystemService.info`